apps/experiments/views/experiment.py

```diff
@@ -943,30 +956,33 @@ def send_invitation(request, team_slug: str, experiment_id: str, session_id: str
 def _record_consent_and_redirect(request, team_slug: str, experiment_session: ExperimentSession):
     # record consent, update status
     experiment_session.consent_date = timezone.now()
-    if experiment_session.experiment.pre_survey:
+    if experiment_session.experiment_version.pre_survey:
         experiment_session.status = SessionStatus.PENDING_PRE_SURVEY
-        redirct_url_name = "experiments:experiment_pre_survey"
+        redirect_url_name = "experiments:experiment_pre_survey"
     else:
         experiment_session.status = SessionStatus.ACTIVE
-        redirct_url_name = "experiments:experiment_chat"
+        redirect_url_name = "experiments:experiment_chat"
     experiment_session.save()
     response = HttpResponseRedirect(
         reverse(
-            redirct_url_name,
-            args=[team_slug, experiment_session.experiment.public_id, experiment_session.external_id],
+            redirect_url_name,
+            args=[team_slug, experiment_session.experiment_version.public_id, experiment_session.external_id],
         )
     )
     return set_session_access_cookie(response, experiment_session)
 
 
 @experiment_session_view(allowed_states=[SessionStatus.SETUP, SessionStatus.PENDING])
 def start_session_from_invite(request, team_slug: str, experiment_id: str, session_id: str):
+    # A session from invite will (for now?) always use the default experiment version
```

Comment:
I think it will always be the default. The only time you can use a non-default is if you're an experiment admin in which case you wouldn't use an invite.

Reply:
Correct. I updated the views to always assume the `experiment_id` is the working experiment (https://github.com/dimagi/open-chat-studio/pull/671/commits/ebaac498dec2fe45ee72c80f7782421a3c55b2a1)